core/types/schema.go

```diff
@@ -1254,21 +1254,35 @@ var (
 	IntType = &DataType{
 		Name: intStr,
 	}
-	TextType = &DataType{
+	IntArrayType = ArrayType(IntType)
+	TextType     = &DataType{
 		Name: textStr,
 	}
-	BoolType = &DataType{
+	TextArrayType = ArrayType(TextType)
+	BoolType      = &DataType{
 		Name: boolStr,
 	}
-	BlobType = &DataType{
+	BoolArrayType = ArrayType(BoolType)
+	BlobType      = &DataType{
 		Name: blobStr,
 	}
-	UUIDType = &DataType{
+	BlobArrayType = ArrayType(BlobType)
+	UUIDType      = &DataType{
 		Name: uuidStr,
 	}
-	Uint256Type = &DataType{
+	UUIDArrayType = ArrayType(UUIDType)
+	// DecimalType contains 0,0 metadata.
```

Comment:
This is just used as a catch-all for any decimal type, where we don't care about precision and scale. For example, if we need to know how to decode some serialized changeset value, we need to know what the underlying type is. Normally, you would use OIDs for this, but since our UUID OIDs will be different on all machines, we have to translate this into Kwil types.

Reply:
I think I understand that this is just an arbitrary *instance* so there's a pointer to something for the map, but I'm still not seeing where these two values (1,0) go.  Is it just anything?  If it were changed to (20,10) in this map key, would anything change anywhere, or are we just putting something arbitrary here.

Reply:
Just putting something arbitrary that is valid. Having an empty array or `0,0` would be invalid. `1,0` is just the lowest valid precision/scale. But yes, anything that is valid would work here and have no negative effects.